cypress/integration/pages/workProgressSpec.ts

```diff
@@ -13,26 +8,34 @@ describe("Work Progress", () => {
     context("when the type field is selected", () => {
       context("when Work Number is selected in dropdown for type", () => {
         before(() => {
-          cy.findByTestId("type").select("SGP/R&D Number");
+          search().within(() =>
+            cy.findByTestId("type").select("SGP/R&D Number")
+          );
         });
         it("shows an input text field", () => {
-          cy.get("[data-testid=valueInput]").should("have.value", "");
+          search().within(() =>
+            cy.get("[data-testid=valueInput]").should("have.value", "")
+          );
         });
       });
       context("when Status is selected in dropdown for type", () => {
         before(() => {
-          cy.findByTestId("type").select("Status");
+          search().within(() => cy.findByTestId("type").select("Status"));
         });
         it("shows a drop down box with value unstarted ", () => {
-          cy.get("[data-testid=valueSelect]").should("have.value", "unstarted");
+          search().within(() =>
+            cy.get("[data-testid=valueSelect]").should("be.visible")
+          );
         });
       });
       context(" when WorkType is selected in dropdown for type", () => {
```

Comment:
Same as above. Could this test it shows the Work Types?

Reply:
Done